src/compiler.adoc

```diff
@@ -821,7 +820,7 @@ link:https://github.com/clojure/clojurescript/wiki#libraries[ClojureScript Wiki]
 To understand dependencies better, we are going to replace the DOM utiltiles of the Google Closure
 library with a third party library for interacting with the DOM.
 
-Let start adding `[prismatic/dommy "1.1.0"]` into the `:dependencies` in the
+Let's start by adding `[prismatic/dommy "1.1.0"]` at the end of the `:dependencies` vector in
```

Comment:
- grammar: Let start adding -> Let's start by adding
- elaborate: where to add dependency so reader may see it immediately